packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```

Comment:
file level ts-ignore, why do we need it?

Reply:
[Same reason](https://github.com/webex/webex-js-sdk/pull/2508#discussion_r1052918338)